torch/csrc/jit/runtime/profiling_graph_executor_impl.h

```diff
@@ -25,6 +25,10 @@ struct ProfilingGraphExecutorImpl : public GraphExecutorImplBase {
     remaining_bailout_depth_.reset();
   }
 
+  bool isOptimized() const override {
```

Comment:
arrggh, all because I for whatever weird reason decided to throw from `getDebugState`.

Reply:
No worries, we can update that in the follow up PR once the throw is gone.